core/src/main/java/jenkins/slaves/JnlpSlaveAgentProtocol.java

```diff
@@ -42,56 +42,20 @@
  * @author Kohsuke Kawaguchi
  * @since 1.467
  */
-@Extension
-@Symbol("jnlp")
 public class JnlpSlaveAgentProtocol extends AgentProtocol {
```

Comment:
```suggestion
public class JnlpSlaveAgentProtocol {
```

so you can get rid of the overrides.

Reply:
Also remove unused imports.

Reply:
(Set `remoting.version` to 3.40 to see that this is necessary.)